### DISCUSSION TITLE
AI趣味圖片:-)

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#109] [General]
在繁忙的期中考週，提供幾張有關於人工智慧的趣味圖給大家放鬆一下吧🤣

![image](https://user-images.githubusercontent.com/90882519/141420346-6f5e88af-3ce0-42fd-a273-826d8b2b44b8.png)
![image](https://user-images.githubusercontent.com/90882519/141420280-5afdc009-4ec5-40c0-a9e7-63b6fb1b6b1e.png)
(AI加...油...，還有很大進步空間!)

![image](https://user-images.githubusercontent.com/90882519/141420850-88077e31-87c2-49a4-a5da-45314160d364.png)
(怕.. AI已經有特異功能了嗎👻)

![image](https://user-images.githubusercontent.com/90882519/141420770-23d4a7e1-ed91-430c-b428-88211c4e2dd0.png)
(似乎有點脫離AI哈哈 但希望自己能多方涉略 最後會是有益的)
p.s. PCA似乎是一種分析法!

#來源都為Google搜尋:AI梗圖，還有很多好笑的在上面 ，無聊時可上去看看喔^^

**Comment by bonny123865** [comment]
那個人臉辨識真的有點恐怖😂
不知道是不是真的有 什麼東西在後面

**Comment by minniewei** [comment]
哈哈哈，現在背後感覺毛毛的都想說要用人臉辨識辨識一下，說不定會有意外的結果。

**Comment by wh20502** [comment]
之前有學過YOLO4我都很害怕拍到什麼奇怪的東西，那張已經不是梗圖是恐怖照片了哈哈

**Comment by ghost** [comment]
搞不好以後去鬼屋探險不是帶靈媒而是帶AI智慧人臉辨識XD

**Comment by handsomeniko** [comment]
看到物件偵測就想到yolo，真的很好用

**Comment by Shiba010** [comment]
會不會以後就有能探測出鬼的AI？！

**Comment by ayakasanma** [comment]
資料分析唯一支持pandas!好用又強大~

**Comment by Lynn12378** [comment]
那個性別不明的太好笑了吧

**Comment by Yunhans** [comment]
dog

**Comment by LanningYang** [comment]
目前只能做差異性較大辨別，但相信未來定能夠更精準的辨識
